lms/templates/wiki/includes/cheatsheet.html

```diff
@@ -1,5 +1,5 @@
 {% load i18n %}
-<section id="cheatsheetModal" class="modal" aria-hidden="true">
+<div id="cheatsheetModal" class="modal" aria-hidden="true">
```

Comment:
@cahrens can you add ```role=dialog```, ```aria-label="Markdown Cheatsheet"```, and  ```aria-modal=true```  here?  ```aria-modal``` is a brand new attribute in aria 1.1 so it may cause an aXe failure.  If that is the case, then lets NOT add it, but create a ticket to create a custom rule that allows it.  At the very least, I want to start including a ```role=dialog``` with either an ```aria-label``` or ```aria-labelledby``` attribute  to all modals now.